internal/gamepad/gamepad.go

```diff
@@ -302,40 +352,43 @@ func (g *Gamepad) IsStandardButtonAvailable(button gamepaddb.StandardButton) boo
 	if gamepaddb.HasStandardLayoutMapping(g.sdlID) {
 		return gamepaddb.HasStandardButton(g.sdlID, button)
 	}
-	return g.native.isStandardButtonAvailableInOwnMapping(button)
+	return g.native.standardButtonInOwnMapping(button) != nil
 }
 
 // StandardAxisValue is concurrent-safe.
 func (g *Gamepad) StandardAxisValue(axis gamepaddb.StandardAxis) float64 {
 	if gamepaddb.HasStandardLayoutMapping(g.sdlID) {
 		return gamepaddb.AxisValue(g.sdlID, axis, g)
 	}
-	if g.native.hasOwnStandardLayoutMapping() {
-		return g.native.axisValue(int(axis))
+	m := g.native.standardAxisInOwnMapping(axis)
+	if m == nil {
+		return 0
 	}
-	return 0
+	return m.Value()*2 - 1
```

Comment:
Sure, if you prefer so - normally I am used to indenting the error flow for consistency (is also part of Google's internal style guide).

Reply:
Just out of curiosity, does Google prefer early-returns with errors, rather than just early returns?

I prefer narrower scopes of variables in general. Both cases are not errors, and I think it is natural that the last item indicates a 'fallback'.